sycl/test-e2e/Matrix/joint_matrix_out_bounds.cpp

```diff
@@ -13,8 +13,11 @@
 // RUN: %{run} %t.out
 
 #include "common.hpp"
-
-constexpr size_t TN = 16;
-constexpr size_t MATRIX_K = 1024 + 24;
-
 #include "joint_matrix_out_bounds_impl.hpp"
+
+int main() {
+  std::cout << "A row major, B row major:\n";
+  test_all<layout::row_major, layout::row_major>();
+  std::cout << "A row major, B packed:\n";
+  test_all<layout::row_major, layout::ext_intel_packed>();
```

Comment:
These tests still do not test transpose of C. Is this something you plan on adding later?

Reply:
No, I did not plan to as part of this task. I'm working on adding support for transpose A and B.

Reply:
Transpose of C is already available but not for the checked variants. Should I create a separate jira for that?

Reply:
I see that some checked shapes are enabled for transpose C, but we don't know if they actually work, since there are no tests. So, I think yes, jira is needed to enable both testing and investigate/enable some shapes, which are not yet enabled.